lib/src/base-styles/_nx-text-input.scss

```diff
@@ -5,33 +5,38 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 
-.nx-text-input {
-  border: $nx-border-dark;
-  border-left-width: 4px;
-  border-radius: 0;
-  box-shadow: none;
+.nx-text-input__input {
+  border: 1px solid #d1d6f0;
+  border-radius: $nx-border-radius;
+  color: $nx-text-color-light;
+  font-size: $nx-font-size;
   height: $nx-form-element-height;
-  padding: 0 8px;
+  padding: 0 $nx-spacing-md;
   width: $nx-form-element-width-normal;
 
+  &:hover {
+    border-color: #2c2c2c;
+  }
+
   &:focus {
+    border-color: #43cbf5;
     outline: none;
   }
 
   &:disabled {
-    background: $nx-off-white;
-    border-color: $nx-grey-3;
-    border-left: 4px double $nx-grey-3;
+    background-color: #e9e9e9;
+    border: solid 1px #949494;
   }
 }
 
 .nx-text-input--long {
   width: $nx-form-element-width-wide;
 }
 
-textarea.nx-text-input {
-  min-height: 64px;
-  padding: 8px;
+textarea.nx-text-input__input {
+  min-height: 220px;
+  padding: $nx-spacing-xs $nx-spacing-md;
+  max-width: 700px;
 }
```

Comment:
`textarea`s are getting `line-height: normal`, need explicit overridding

Reply:
> `textarea`s are getting `line-height: normal`, need explicit overridding

Both inputs had lost their fonts and were pulling the line-height from _user agent_. I restored both settings here: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/7f3a465a2bea91a07b2f52357e48f986d4d19806